addons/website/static/src/js/content/menu.js

```diff
@@ -540,7 +540,7 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
      * @private
      */
     _onDropdownShow: function (ev) {
-        const $dropdown = $(ev.target).closest('.dropdown');
+        const $dropdown = $(ev.target).closest('.dropdown, .dropup');
```

Comment:
I am surprise they don't have a common selector to target a dropdown regardless if it is a dropup, dropdown, dropright or dropleft, but that's how it is.